cw/simple_capture_aes_sca.py

```diff
@@ -238,16 +265,37 @@ def abort_handler_during_loop(this_project, sig, frame):
 
     # Create and show test plot ------------------------------------------------
     # Use this plot to check for clipping and adjust gain appropriately
+    if CWLib:
+        proj_waves = project.waves
+    else:
+        proj_waves = project.get_waves()
```

Comment:
nit: could we pls load just #plot_traces here? :)

Super nice the whole thing!